packages/forklift-console-plugin/src/modules/Providers/views/create/components/OvirtProviderCreateForm.tsx

```diff
@@ -88,19 +91,16 @@ export const OvirtProviderCreateForm: React.FC<OvirtProviderCreateFormProps> = (
         label={t('URL')}
         isRequired
         fieldId="url"
-        helperText={t(
-          'Enter the API engine URL for the Red Hat Virtualization (RHV) provider. Ensure it includes the "ovirt-engine/api" path, e.g., https://rhv.com/ovirt-engine/api.',
-        )}
         validated={state.validation.url}
-        helperTextInvalid={state.validation.urlHelperTextInvalid}
+        helperText={state.validation.urlHelperText}
+        helperTextInvalid={state.validation.urlHelperText}
       >
         <TextInput
           isRequired
           type="text"
           id="url"
           name="url"
           value={url}
-          validated={state.validation.url}
```

Comment:
why did you drop the error / warning notification ?

Patternfly suggest informing users about erros and warnings, see:
https://www.patternfly.org/components/forms/text-input#invalid

Reply:
I didn't drop it. It's still working since it's set for the `FormGroup` https://github.com/kubev2v/forklift-console-plugin/pull/744/files#diff-075b2af748cc5c8407a5cd156371011523517ef954d89d2bac7d9d5c72d9fe97L94 so no need to duplicate it.

Reply:
In the screenshot you provided the icon for warning is missing, AFAIU this change was triggered by removing the validated field, can you re check ?

Reply:
You are right, my mistake. The icon within the text field was removed due to this change. I'll add it.